samples/GuiSamples.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
This file could probably be named `TerraFX.Interop.Windows.Samples.sln` and be in the root next to `TerraFX.Interop.Windows.sln` for discoverability